packages/parser-typescript-config/src/parser.ts

```diff
@@ -39,6 +49,54 @@ export default class TypeScriptConfigParser extends Parser<TypeScriptConfigEvent
         return validationResult;
     }
 
+    private compilerOptionsExists(schema: any) {
+        return schema.definitions &&
+            schema.definitions.compilerOptionsDefinition &&
+            schema.definitions.compilerOptionsDefinition.properties &&
+            schema.definitions.compilerOptionsDefinition.properties.compilerOptions;
+    }
+
+    private typeAcquisitionExists(schema: any) {
+        return schema.definitions &&
+            schema.definitions.typeAcquisitionDefinition &&
+            schema.definitions.typeAcquisitionDefinition.properties &&
+            schema.definitions.typeAcquisitionDefinition.properties.typeAcquisition;
+    }
+
+    private async updateSchema() {
+        let schemaStat: fs.Stats;
+        const now = Date.now();
+
+        try {
+            schemaStat = await promisify(fs.stat)(this.schemaPath);
+
+            const modified = new Date(schemaStat.mtime).getTime();
+
+            if (now - modified > oneDay) {
+                debug('TypeScript Schema is older than 24h.');
+                debug('Updating TypeScript Schema');
+                const res = JSON.parse(await requestAsync('http://json.schemastore.org/tsconfig'));
+
+                if (this.compilerOptionsExists(res)) {
+                    res.definitions.compilerOptionsDefinition.properties.compilerOptions.additionalProperties = false;
+                }
+
+                if (this.typeAcquisitionExists(res)) {
+                    res.definitions.typeAcquisitionDefinition.properties.typeAcquisition.additionalProperties = false;
+                }
+
+                this.schema = res;
+
+                await writeFileAsync(this.schemaPath, JSON.stringify(res, null, 2));
+            }
+        } catch (e) {
```

Comment:
There are 2 main action that can fail:

1. `fs.stat` for the local file
1. Download the remote file

I think I'll split the `try/catch` in 2 so we can have better debugging output. They could even be separate functions so we don't have a `try/catch` in this method.